src/cli/project.py

```diff
@@ -232,55 +291,41 @@ def up(ctx, project, organization, ingress, provider, workers, **kwargs):
     Start/Resume a project cluster.
     """
 
-    # GraphQL
     try:
         graph_ql = GraphQL(authentication=ctx.auth)
-        data = graph_ql.query(
-            """
-            {
-                allProjects {
-                    results {
-                        id
-                        title
-                        clusterSettings {
+        if not organization:
```

Comment:
Is the else part really necessary? Cant we solve this with the GraphQL filter `oraganization_id`?

![Screenshot from 2021-07-30 13-09-31](https://user-images.githubusercontent.com/6874607/127644705-42a5082d-8ff9-4e0a-a4f3-f90c0e47ba07.png)

Reply:
I think it should work with `oroganization_id = None`

Reply:
actually it works fine with the organization_id as well, it seemed to me as more clear or self-explanatory having an if branch here. But of course having only one query reduces complexity, so I will refactor it to one query.